test/rules/function-body-length/default/test.ts.lint

```diff
@@ -0,0 +1,302 @@
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+    ];
+};
+
+const functionExpressionLong = function () {
+                               ~~~~~~~~~~~~~
+    return [
+~~~~~~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+        1,
+~~~~~~~~~~
+    ];
+~~~~~~
+};
+~ [Maximum function expression body length exceeded: 101 lines instead of <= 100.]
```

Comment:
Surely there must be a better way to do this.

Reply:
I actually quite like the unit testing system here 😊

Reply:
300 lines of `1`s 😁